.github/workflows/ts-check.yml

```diff
@@ -14,7 +14,8 @@ jobs:
         with:
           node-version: 22.x
           registry-url: 'https://registry.npmjs.org'
+          cache: 'npm'
       - name: Install dependencies 📦
-        run: npm install
+        run: npm ci
```

Comment:
What does this change do? Why from install to ci? What does ci do?

Reply:
The "`ci`" [is short for "clean install"](https://docs.npmjs.com/cli/v11/commands/npm-ci). It's basically `npm install` but with a couple of key differences:

- It totally wipes out any existing `node_modules` folder (which shouldn't really apply to these jobs).
- If `package-lock.json` is out of date (relative to what's specified in `package.json`), it will raise an error rather than silently updating it.

The NPM documentation recommends using `npm ci` in CI builds.

Reply:
Looks like I missed updating one workflow, though.